.c9/runners/c.run

```diff
@@ -0,0 +1,18 @@
+// Create a custom Cloud9 runner - similar to the Sublime build system
+// For more information see http://docs.c9.io:8080/#!/api/run-method-run
+{
+    "script": [
+        "base=$file_base_name",
+        "export MODEL=$(awk '{print $NF}' /proc/device-tree/model)",
+        "export TARGET=${base%%.*}",    
+        "export   PRUN=`awk '$1 ~ /^#define/ && $2 ~ /PRUN/ {print $3}' $file_name`",    
```

Comment:
Interesting. If it isn’t defined, can a regular ARM app be built? Maybe the term should be a bit more generic so I can build OpenCV/TIDL/MJPG-streamer apps the same way?

Reply:
If PRUN isn't defined you could set different flags in the Makefile, or call a whole other Makefile.

Reply:
PRUN doesn't seem like a generic term to define a target processor. The same methodology could easily be applied for C66 or M4 programs and probably should be. Maybe another variable name would make sense? Why don't we just make it part of the filename like file.pru1_1.c?

Not sure why I didn't add the extra period before.